### PR TITLE
Update version to v0.5.3

### PR DESCRIPTION
**Pull request by maltekliemann**

### CODE REVIEW

File: runtime/battery-station/src/lib.rs [hunk 1]
```diff
@@ -104,8 +104,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("zeitgeist"),
     impl_name: create_runtime_str!("zeitgeist"),
     authoring_version: 1,
-    spec_version: 55,
-    impl_version: 1,
+    spec_version: 56,
+    impl_version: 0,
```
Comment (Chralt98) [inline runtime/battery-station/src/lib.rs:108]: Why you did change the `impl_version`? Can't we just leave it as the default, which is one?

File: runtime/zeitgeist/src/lib.rs [hunk 2]
```diff
@@ -93,8 +93,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("zeitgeist"),
     impl_name: create_runtime_str!("zeitgeist"),
     authoring_version: 1,
-    spec_version: 55,
-    impl_version: 1,
+    spec_version: 56,
+    impl_version: 0,
```
Comment (Chralt98) [inline runtime/zeitgeist/src/lib.rs:97]: Why you did change the `impl_version`? Can't we just leave it as the default, which is one?

File: runtime/battery-station/src/lib.rs (hunk 1 above)
Comment (Chralt98) [reply runtime/battery-station/src/lib.rs:108]: Oh I didn't read `This number can be reverted to 0 after a [spec_version](https://paritytech.github.io/polkadot-sdk/master/cumulus_test_client/sc_executor/struct.RuntimeVersion.html#structfield.spec_version) bump.`

File: runtime/zeitgeist/src/lib.rs (hunk 2 above)
Comment (Chralt98) [reply runtime/zeitgeist/src/lib.rs:97]: Oh I didn't read `This number can be reverted to 0 after a [spec_version](https://paritytech.github.io/polkadot-sdk/master/cumulus_test_client/sc_executor/struct.RuntimeVersion.html#structfield.spec_version) bump.`

**Review by Chralt98 — APPROVED** [review]

File: runtime/battery-station/src/lib.rs (hunk 1 above)
Comment (maltekliemann) [reply runtime/battery-station/src/lib.rs:108]: Yeah, it's fairly irrelevant, but it kind of bothered me that it's at 1 for no (?) reason.

**Review by Chralt98 — APPROVED** [review]

**Pull request merged**